Completion/X/Command/_kdeconnect-cli

```diff
@@ -0,0 +1,35 @@
+#compdef kdeconnect-cli
+
+_kdeconnect-cli() {
+  _kdeconnect-cli-devices-id() {
+    compadd $(kdeconnect-cli --list-available --id-only)
+  }
+
+  _arguments \
+    '(-l --list-devices)'{-l,--list-devices}'[List all devices]' \
+    '(-a --list-available)'{-a,--list-available}'[List available (paired and reachable) devices]' \
+    '--id-only[Make --list-devices or --list-available print only the devices id, to ease scripting]' \
+    '--refresh[Search for devices in the network and re-establish connections]' \
+    '--pair[Request pairing to a said device]' \
```

Comment:
Which device is the "said" device? When has it been said? Perhaps this has been copied from kdeconnect-cli's help output but I think the wording would be clearer without the word "said". Especially for non-native English speakers.